blocks/extra-large-manual-promo-block/features/extra-large-manual-promo/default.jsx

```diff
@@ -131,7 +127,6 @@ const ExtraLargeManualPromo = ({ customFields }) => {
 									component={Link}
 									condition={linkURL}
 									href={formatURL(linkURL)}
-									className={`${BLOCK_CLASS_NAME}__img`}
```

Comment:
We'll still need this class.

Reply:
@vgalatro Fixed it in [a622ba2](https://github.com/WPMedia/arc-themes-blocks/pull/1690/commits/a622ba26042fa29d7ded26bda4f557df321862c0)